pkg/noobaaaccount/noobaaaccount.go

```diff
@@ -72,6 +72,7 @@ func CmdUpdate() *cobra.Command {
 		Run:   RunUpdate,
 	}
 	cmd.Flags().String("new_default_resource", "", "(must be provided) update the default resource on which new buckets will be created")
+	cmd.Flags().Bool("force_md5_etag", false, "This flag enables md5 etag calculation for account")
```

Comment:
That's a good point. If the flag is not set we should not update it at all. we should only update if we get a value.
I think that you can use this to check it:
```go
if cmd.Flags().Changed("force_md5_etag")
```

Reply:
you need to change the type of `ForceMd5Etag` here to `*bool` instead of `bool` to make it optional

https://github.com/achouhan09/noobaa-operator/blob/1dbd75fcc46b8337f9bf4abfc5d18570c09626f6/pkg/nb/types.go#L579

Reply:
Done, Thanks for suggestions.

We cannot make ForceMd5Etag to `*bool` as in commands ForceMd5Etag was not global and we are fetching it from command line flag in every operation(create/update). So, it doesn't create any difference I think. Pls let me know if I am wrong here.

Reply:
I'm not sure I understand the comment. 
for account update  operation it's possible that the user want to update certain fields and level the ForceMd5Etag as is. in the current implementation, the update will set `ForceMd5Etag=false` if it was not passed as a flag.

Reply:
I have updated the logic, PTAL.